<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>104</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>104</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="wordCountSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="wordLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateEnglishButton">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>150</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordCountLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of words</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordLengthLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length of each word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateRussianButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Russian</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
